### PR TITLE
fixed issue with backslash and corrected typo

### PR DESCRIPTION
**Pull request by Timothy248**
fixed issues with importing on windows (replace backslash with forward slash) and corrected a typo

### CODE REVIEW

File: src/compile.ts [hunk 1]
```diff
@@ -161,5 +161,8 @@ function getRelativeImportPath(
   const relativeReference =
     relativePath.length === 0 || !relativePath.startsWith('.') ? './' : '';
 
-  return relativeReference + path.join(relativePath, fileToRequire);
+  const joinedPath = relativeReference + path.join(relativePath, fileToRequire)
+  
+  // replace all backslashes with forward slashes to fix issue with importing on windows
+  return joinedPath.replace("\\", "/");
```
Comment (cometkim) [inline src/compile.ts:167]: This only replaces the first match

**Review by cometkim — CHANGES_REQUESTED** [review]

File: src/compile.ts (hunk 1 above)
Comment (Timothy248) [reply src/compile.ts]: oh shoot, i fixed it!

**Comment by cometkim** [comment]
Umm... could you please avoid including unnecessary format changes?

**Comment by Timothy248** [comment]
I'm so sorry! This is my first time contributing.. Reverted the formatting. Please excuse my mistakes😅

**Comment by cometkim** [comment]
No problem, it's a pretty common mistake

Thanks for reporting this

**Comment by Timothy248** [comment]
Thanks :) Can you merge this? or does it need other changes?

**Review by cometkim — APPROVED** [review]

File: src/compile.ts [hunk 2]
```diff
@@ -161,5 +161,8 @@ function getRelativeImportPath(
   const relativeReference =
     relativePath.length === 0 || !relativePath.startsWith('.') ? './' : '';
 
-  return relativeReference + path.join(relativePath, fileToRequire);
-}
+  const joinedPath = relativeReference + path.join(relativePath, fileToRequire)
+  
+  // replace all backslashes with forward slashes to fix issue with importing on windows
+  return joinedPath.replaceAll("\\", "/");
```
Comment (cometkim) [inline src/compile.ts:167]: Hmm, well I guess maybe this doesn't work where there are legitimate escape paths, e.g. if the workspace path has spaces: `/My\ Workspace/project` ?

Comment (cometkim) [reply src/compile.ts:167]: Ah, nevermind, `\\` is escaped one

**Pull request merged**